integrations/lib/multiwoven/integrations/destination/http/client.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+require "base64"
+module Multiwoven
+  module Integrations
+    module Destination
+      module Http
+        include Multiwoven::Integrations::Core
+        class Client < DestinationConnector
+          MAX_CHUNK_SIZE = 10
+          def check_connection(connection_config)
+            connection_config = connection_config.with_indifferent_access
+            destination_url = connection_config[:destination_url]
+            request = Multiwoven::Integrations::Core::HttpClient.request(
```

Comment:
@AnthonyR-AI2  how are we confirming the credentials are valid by just sending an options request without any body ?

Reply:
Addressed this in my previous comment, this will change and will not be an options request.